### PR TITLE
JBPM-5595: Upgrade lienzo-core to version 2.0.286-RELEASE.

### PR DESCRIPTION
**Pull request by romartin**
Hey @manstis @hasys 
And here are the changes for the `lienzo-core` version upgrade. Basically it's about removing the fixes that already present in the new `lienzo-core` release.
Depends on https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/416
Thanks!
Roger

### CODE REVIEW

**Comment by romartin** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
@etirelli Do you know of [DMN](https://kie-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/kie-wb-common-pullrequests/984/console) compilation failures?

**Comment by manstis** [comment]
Jenkins please retest this.

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/wires/CaseManagementDockingAndContainmentControlImpl.java [hunk 1]
```diff
@@ -22,15 +22,21 @@
 import com.ait.lienzo.client.core.shape.wires.WiresManager;
 import com.ait.lienzo.client.core.shape.wires.WiresShape;
 import com.ait.lienzo.client.core.shape.wires.WiresUtils;
+import com.ait.lienzo.client.core.shape.wires.handlers.impl.WiresDockingAndContainmentControlImpl;
 import com.ait.lienzo.client.core.shape.wires.picker.ColorMapBackedPicker;
 import com.ait.lienzo.client.core.types.Point2D;
 import com.ait.tooling.nativetools.client.collection.NFastArrayList;
-import org.kie.workbench.common.stunner.lienzo.wires.handlers.impl.WiresDockingAndContainmentControlImpl;
 
 public class CaseManagementDockingAndContainmentControlImpl extends WiresDockingAndContainmentControlImpl {
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/wires/CaseManagementDockingAndContainmentControlImpl.java]: Sorry @romartin Your (I assume) conflict resolution of this class reverts it to the state it was at before https://github.com/kiegroup/kie-wb-common/pull/724

Can you please copy the code from [here](https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/wires/CaseManagementDockingAndContainmentControlImpl.java) and correct any imports - you should not need to do anything else.

Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/wires/CaseManagementDockingAndContainmentControlImpl.java]: Hey @manstis , I think this is correct now, consider that I have removed the custom control implementation from `lienzo-extensions`, so there is no `org.kie.workbench.common.stunner.lienzo.wires.handlers.impl` package now. You should be inheriting directly from the lienzo ones, isn't it?

**Comment by etirelli** [comment]
@manstis I am now aware of any compilation failure with DMN. I just tried master again, and I don't see any failure.

**Comment by romartin** [comment]
Hey @manstis , yeah sorry, I reverted some changes by error.. added a new commit, hope all good now, I have re-tested the showcase and it seems all fine.
Thanks!

**Comment by manstis** [comment]
Jenkins please retest this.

**Pull request merged**